README.md

```diff
@@ -63,13 +63,13 @@ osc sr systemsmanagement:SCC:RMT yast2-rmt openSUSE:Factory --no-cleanup
 To make the initial submit for a new SLES version:
 
 ```bash
-osc -A https://api.suse.de sr Devel:SCC:RMT yast2-rmt SUSE:SLE-15:GA --no-cleanup
+osc -A https://api.suse.de sr Devel:SCC:RMT yast2-rmt SUSE:SLE-15 --no-cleanup
```

Comment:
That should also say `SUSE:SLE-15:Update`.
Also I'm not sure why we have a separate section about initial submit -- that's never going to happen again :-)
`sr` command will automatically create a MR once the project already exists, but `sr` automagically fills the comment for the MR with a changelog from the changes file.

Reply:
fixed, thank you!